test/lib/ufe/testObject3d.py

```diff
@@ -16,6 +16,9 @@
 # limitations under the License.
 #
 
+# Python 2 and 3: forward-compatible
+from builtins import range
+
```

Comment:
I don't think you should need this. `range()` exists in Python 2, it's just not a generator like it is in Python 3, and for the purposes of this test, the difference shouldn't matter.

Reply:
Right, I don't think it is a biggie but will get rid of  line 20 in another PR.

Reply:
Please see d10a527af2a42983e9d76263c3528ee1236d8792